src/main/java/org/opensearchmetrics/metrics/release/ReleaseInputs.java

```diff
@@ -17,12 +17,13 @@ public enum ReleaseInputs {
     VERSION_2_16_0("2.16.0", "closed", "2.16", false),
     VERSION_2_17_0("2.17.0", "closed", "2.17", true),
     VERSION_2_18_0("2.18.0", "closed", "2.18", true),
+    VERSION_2_19_0("2.19.0", "open", "2.x", true),
```

Comment:
When does this change to `2.19`?

Reply:
when `closed` aka when the release is launched.

Reply:
I meant `2.x` -> `2..19`

Reply:
When we create a new release branch Sayali.